Source/Core/Core/ConfigLoaders/BaseConfigLoader.cpp

```diff
@@ -53,7 +54,19 @@ class INIBaseConfigLayerLoader final : public Config::ConfigLayerLoader
         const IniFile::Section::SectionMap& section_map = section.GetValues();
 
         for (const auto& value : section_map)
-          config_section->Set(value.first, value.second);
+        {
+          const Config::ConfigLocation location{system.first, section_name, value.first};
```

Comment:
I don't have an ideal solution.  Maybe it's own file:  Source/Core/Core/ConfigLoaders/Common/ConvertLegacyEFBScale ?  Or just "common" and we could move other stuff in as appropriate?